docs/current/cookbook/snippets/environment-variables/index.ts

```diff
@@ -0,0 +1,30 @@
+import Client, { connect } from "@dagger.io/dagger"
```

Comment:
```suggestion
import { connect } from "@dagger.io/dagger"
```

Client is used nowhere. You can delete it. Thanks

Reply:
Or instead of deleting it, type the callback argument in connect.